spec/models/avatar_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'carrierwave/test/matchers'
+
+describe AvatarUploader do
+  include CarrierWave::Test::Matchers
+
+  before do
+    AvatarUploader.enable_processing = true
+    @uploader = AvatarUploader.new(@user, :avatar)
+
+    File.open(path_to_file) do |f|
+      @uploader.store!(f)
+    end
+  end
+
+  after do
+    AvatarUploader.enable_processing = false
+    @uploader.remove!
+  end
+
+  context 'the thumb version' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.